clients/python/tests/test_client.py

```diff
@@ -72,10 +72,10 @@ def test_register_existing_version(client: ModelRegistry):
         "model_format_version": "test_version",
         "version": "1.0.0",
     }
-    client.register_model(**params)
+    client.register_model(**params, metadata=None)
 
     with pytest.raises(StoreError):
-        client.register_model(**params)
+        client.register_model(**params, metadata=None)
```

Comment:
question: is there a reason why we needed to be explicit of setting it to None here? 🤔

Reply:
sorry I'm not sure I follow 100%, isn't the default None already? 🤔 

https://github.com/kubeflow/model-registry/blob/7506eafbb18b60e52fcf63c27e09ee3c27f64be1/clients/python/src/model_registry/_client.py#L141

of which hints we're missing please? For awareness.

Reply:
The type checker complains that it doesn't know whether the `params` dict has an appropriate key for `metadata`. So if you specify that, instead of using the `params` dict, the type checker knows you're specifying a valid entry for `metadata`. simple as that :)

Reply:
Aaa on this call side! Sorry I missed it